assets/less/index.less

```diff
@@ -53,7 +58,7 @@
 
 #home {
 	background: @gray-lighter center center/100% url('../img/stock/home-bg.jpg');
-	height: 640px;
+	height: 100vh;
```

Comment:
We will need to add a media query. Background repeats when the device is width is low.